pkg/manager/manager.go

```diff
@@ -64,13 +64,6 @@ type Manager interface {
 	// election was configured.
 	Elected() <-chan struct{}
 
```

Comment:
Move this into the option struct. I think we don't lose any features as metrics handler could only be added until the manager was started anyway